distributor-api-v1/operations.md

```diff
@@ -404,15 +404,35 @@ TBC
 
 A localized text is an object of the property values localized into languages supported by hotel, indexed by language codes.
 
-#### Payment Gateway   <a id="payment-gateway"></a>
+#### PaymentGateway   <a id="payment-gateway"></a>
 
 If the hotel does not use any payment gateway, the value is null. If it does, then you should use a specific api call and the gateway’s library to encode credit card data. The main purpose of a payment gateway is to securely obtain credit card of the customer before a reservation is created. You can decide not to support any of them and just ignore it, in which case reservations are created with note about missing credit card.
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `PaymentGatewayType` | string | required | Type of the payment gateway \(`Adyen`, `Stripe` or `PciProxy`\). |
+| `PaymentCardStorageType` | string [PaymentCardStorageType](operations.md#payment-card-storage-type) | required | Type of the payment card storage used by enterprise. |
 | `IsMerchant` | boolean | required | Whether the gateway is processed through Mews Merchant or not. |
-| `SupportedCreditCardTypes` | array of string | required | The list of supported credit cards, should be used to enhance UX. |
+| `SupportedCreditCardTypes` | string [CreditCardType](operations.md#credit-card-type) | required | The list of supported payment cards, should be used to enhance UX. |
```

Comment:
How can property of type string contain "list of supported..."?

Reply:
It is string that contains enum value. Same as here:
https://app.gitbook.com/@mews-systems/s/connector-api/operations/customers#gender

Reply:
But the description cannot be "The **list** of supported payment cards."